src/migrations/1719945719397_add-new-event-types.ts

```diff
@@ -0,0 +1,11 @@
+import type { MigrationBuilder } from "node-pg-migrate"
+import { EventType } from "../shared/types/events"
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.addTypeValue({name: 'event_type'}, EventType.ProposalFinished)
+  pgm.addTypeValue({name: 'event_type'}, EventType.VestingCreated)
+}
+
+export async function down(): Promise<void> {
+  return
```

Comment:
shouldn't you remove those event types?

Reply:
It's not recommended to do that ([check here](https://supabase.com/docs/guides/database/postgres/enums#removing-enum-values:~:text=%27content%27%3B-,Removing%20enum%20values,-%23))